UI/Web/src/app/series-detail/series-detail.component.ts

```diff
@@ -261,6 +261,12 @@ export class SeriesDetailComponent implements OnInit, OnDestroy {
 		}
 	}
 
+	getHeight(): string {
```

Comment:
Instead of a getHeight, let's use a getter method:

get ScrollingBlockHeight() {
  return string here;
}

And like I mentioned in other comment, let's use a viewchild.

@ViewChild('scrollingBlock') scrollingBlockElement!: ElementRef<HtmlDivElement>;

When using viewchild, it will be selected on ngAfterViewInit, so you should always do a check if it's set.